test/tstDetailsTreeConstruction.cpp

```diff
@@ -153,13 +163,14 @@ void generateHierarchy(Primitives primitives, MortonCodes sorted_morton_codes,
       "Testing::indices", n);
   ArborX::iota(space, permutation_indices);
 
+  Test::FakeBoundingVolume bounds;
```

Comment:
Suggested to deduce the bounding volume type instead of spelling it out.  See new revision of aprokop/ArborX#16